openmc/lib/math.py

```diff
@@ -205,37 +209,45 @@ def rotate_angle(uvw0, mu, phi=None):
 
     """
 
-    uvw0_arr = np.array(uvw0, dtype=np.float64)
+    if prn_seed is None:
+        prn_seed = getrandbits(64)
 
+    uvw0_arr = np.array(uvw0, dtype=np.float64)
     if phi is None:
-        _dll.rotate_angle_c(uvw0_arr, mu, None)
+        _dll.rotate_angle_c(uvw0_arr, mu, None, c_uint64(prn_seed))
     else:
-        _dll.rotate_angle_c(uvw0_arr, mu, c_double(phi))
+        _dll.rotate_angle_c(uvw0_arr, mu, c_double(phi), c_uint64(prn_seed))
+
     uvw = uvw0_arr
 
     return uvw
 
 
-def maxwell_spectrum(T):
+def maxwell_spectrum(T, prn_seed=None):
     """ Samples an energy from the Maxwell fission distribution based
     on a direct sampling scheme.
 
     Parameters
     ----------
     T : float
         Spectrum parameter
+    prn_seed : int
+        PRNG seed; if None, one will be generated randomly
 
     Returns
     -------
     float
         Sampled outgoing energy
 
     """
-
-    return _dll.maxwell_spectrum(T)
+    
+    if prn_seed is None:
+        prn_seed = randint(0,10000000)
```

Comment:
Sorry, forgot to comment on this one before -- also change to `getrandbits` here. Also, it's probably safer to use 63 bits since we're using a 63-bit LCG (the mask probably takes care of the extra bit, but just in case).